After updating the epson-inkjet-escpr package, some architectures would not build correctly due to needing large file support.
Adding this line to the config file adds large file support when building the image. 

Original uprev bug can be found here: https://issuetracker.google.com/276773064
Learn more about LFS here: https://issuetracker.google.com/201531268

--- a/configure.ac
+++ b/configure.ac
@@ -33,6 +33,7 @@ AM_INIT_AUTOMAKE([gnu 1.7])
 
 dnl Checks for programs.
 AC_PROG_CC
+AC_SYS_LARGEFILE
 AC_C_CONST
 AC_PROG_CPP
 AC_PROG_INSTALL
